sorc/gefs_ensstat.fd/EPDF.f

```diff
@@ -240,10 +240,10 @@ function epdf(ef,ew,n,qq,ictl)
       do i = 1, m
        if (qq.lt.q(i)) then
         k=i-1
-        goto 101
+        exit
```

Comment:
This is functionally different than the control. Are we sure this performs as desired?

Reply:
Nevermind. Not used to dealing with FORTRAN loops.